app/i18n/en/conf.php

```diff
@@ -21,6 +21,7 @@
 			'publication_date' => 'Date of publication',
 			'related_tags' => 'Article tags',
 			'sharing' => 'Sharing',
+			'display_authors' => 'Display Authors',
```

Comment:
Could you also add a screenshot of the config? I'm thinking just "authors" would probably be fine.

Reply:
I agree, just "authors" would be better.

![image](https://user-images.githubusercontent.com/2742474/62852930-24aa7c80-bceb-11e9-91d8-199912e00c08.png)